demos/_unit-test/lookup-virtual-page.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * Behat testing.
```

Comment:
delete this comment, implied by filename

Reply:
Done